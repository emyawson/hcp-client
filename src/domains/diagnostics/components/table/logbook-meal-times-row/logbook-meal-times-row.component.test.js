import { shallow } from 'enzyme';
import React from 'react';

import { LogbookMealTimesRow } from '.';

const mockProps = {
  mealTimes: {
    BREAKFAST: {
      hasBeforeAndAfterIntervals: true,
      measurements: [],
      numberOfRowsWithContent: 2,
    },
  },
  numberOfRows: 3,
};

describe('LogbookMealTimesRow test suite', () => {
  test('it renders correctly', () => {
    const wrapper = shallow(<LogbookMealTimesRow {...mockProps} />);
    expect(wrapper).toMatchSnapshot();
  });
});
