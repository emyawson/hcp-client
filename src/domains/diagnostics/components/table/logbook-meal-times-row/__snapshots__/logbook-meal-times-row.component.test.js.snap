// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogbookMealTimesRow test suite it renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    mealTimes={
      Object {
        "BREAKFAST": Object {
          "hasBeforeAndAfterIntervals": true,
          "measurements": Array [],
          "numberOfRowsWithContent": 2,
        },
      }
    }
    numberOfRows={3}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Array [
    <Unknown
      alternateFill={true}
      day={undefined}
      hasBeforeAndAfterIntervals={true}
      mealTimeMatrix={Array []}
      mouseClickHandler={[Function]}
      numberOfRowsWithContent={2}
      numberOfTotalRows={3}
      width="15rem"
    />,
  ],
  Symbol(enzyme.__nodes__): Array [
    Array [
      <Unknown
        alternateFill={true}
        day={undefined}
        hasBeforeAndAfterIntervals={true}
        mealTimeMatrix={Array []}
        mouseClickHandler={[Function]}
        numberOfRowsWithContent={2}
        numberOfTotalRows={3}
        width="15rem"
      />,
    ],
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
